### PR TITLE
Arm and other cleanups

### PR DESCRIPTION
**Pull request by nullr0ute**
Some changes needed for UEFI support on ARMv7 as well as some small package updates

### CODE REVIEW

File: share/templates.d/99-generic/runtime-install.tmpl [hunk 1]
```diff
@@ -68,9 +68,6 @@ installpkg plymouth
 ## extra dracut modules
 installpkg anaconda-dracut dracut-network dracut-config-generic dracut-fips
 
-## import-state.service for switchroot
-installpkg initscripts
```
Comment (bcl) [inline share/templates.d/99-generic/runtime-install.tmpl]: NAK on this line. See https://bugzilla.redhat.com/show_bug.cgi?id=1615332

File: share/templates.d/99-generic/runtime-install.tmpl [hunk 2]
```diff
@@ -30,11 +30,15 @@ installpkg glibc-all-langpacks
 
 ## arch-specific packages (bootloaders etc.)
 %if basearch == "aarch64":
-    installpkg efibootmgr grub2-efi-aa64-cdboot grubby shim-aa64
```
Comment (bcl) [inline share/templates.d/99-generic/runtime-install.tmpl]: Are you sure grubby doesn't need to be here? Have you tested this template?

File: share/templates.d/99-generic/runtime-install.tmpl (hunk 1 above)
Comment (nullr0ute) [reply share/templates.d/99-generic/runtime-install.tmpl]: There was no comment from the initscripts maintainer as to why it was needed, and there's also no documentation there as to what broke, it just says "some network configuration use cases are broken" which isn't useful.

Can you merge the arm patch without the other one?

File: share/templates.d/99-generic/runtime-install.tmpl (hunk 2 above)
Comment (nullr0ute) [reply share/templates.d/99-generic/runtime-install.tmpl]: grubby is explicitly there for all architectures on line 16 above that

**Comment by bcl** [comment]
Picked the arm patch to master.

**Pull request closed without merging**